### ISSUE TITLE
the pretrained model test results not good

### ISSUE THREAD

**Issue by semchan** [csjliang/LPTN#13]
I used your pretrained model to test the images on FiveK datasets, but some images will generate "Ghost shadow" on the images (eg. the middle image as below). Would you please help to fix this issue? or can you give me some guide to avoid this? thanks a lot.
![4501_LPTN_FiveK_480p](https://user-images.githubusercontent.com/20419555/120738666-cfc0a180-c522-11eb-87f7-ec4edf72f91b.png)

**Comment by csjliang** [comment]
Hi, thanks for your question. 
Sure, there are a small number of images in the FiveK test set that will suffer artifacts. This is caused by the GAN training in the unsupervised scenario and is currently a challenging problem in research. To totally avoid this, paired training is always employed in practice using only the MSE loss. If you don't want to use paired training data, you might need to tune the weight of MSE and GAN losses (decrease the loss of GAN) to find a good trade-off between translation effects and fidelity. Thanks.

**Comment by semchan** [comment]
> Hi, thanks for your question.
> Sure, there are a small number of images in the FiveK test set that will suffer artifacts. This is caused by the GAN training in the unsupervised scenario and is currently a challenging problem in research. To totally avoid this, paired training is always employed in practice using only the MSE loss. If you don't want to use paired training data, you might need to tune the weight of MSE and GAN losses (decrease the loss of GAN) to find a good trade-off between translation effects and fidelity. Thanks.

Thank you very much for your reply. I read your paper and found that you have also collected new data sets with higher resolution. When will it be open source?

**Comment by csjliang** [comment]
The data will be released in about a month. Thanks for your attention.

**Comment by semchan** [comment]
> The data will be released in about a month. Thanks for your attention.

Thanks a lot. BTW, the train dataset in FiveK is not paired? am I correct?

**Comment by csjliang** [comment]
In our paper, we train photo retouching models using unpaired data (2250 inputs and 2250 targets). The FiveK dataset has paired data, i.e., 4500 pairs of training data with both inputs and targets.

**Comment by csjliang** [comment]
I will close this issue if you have no further questions.
